### DISCUSSION TITLE
Application setup > Enable OBS Studio...

### DISCUSSION THREAD

**Discussion by e-Feitosa** [sircharlo/meeting-media-manager#1359] [Ideas] [answered]
When we install M3 for the first time and we are going to configure the OBS Studio option, when we click to activate the item below:
![image](https://user-images.githubusercontent.com/90984283/216783556-49928325-85c0-4d30-b1c1-fa849f423422.png)

The option to use plugin 4 is already active:
![image](https://user-images.githubusercontent.com/90984283/216783584-88c67e17-d69f-4867-9bda-371b1c06f62b.png)

However, plugin version 5 is already native in the current version of OBS Studio:
![image](https://user-images.githubusercontent.com/90984283/216783659-1f4c7462-879d-4b07-aabe-f3966aed9392.png)

The **suggestion** is to leave this option disabled:
![image](https://user-images.githubusercontent.com/90984283/216783736-6fd51e96-66d3-49c8-bcc2-5cbd8a21c348.png)

**Comment by mtdvlpr** [comment, marked as answer]
Hi @e-Feitosa, this was initially done to prevent accidentally breaking the obs connection when upgrading to the version that supported v5 for the first time. But I agree that now there's been enough time to update, so the default will be changed in the next release :)
